nc-game-front-end/src/components/AllUsers.jsx

```diff
@@ -0,0 +1,30 @@
+
+import { fetchUsers } from "../api";
+import UserCard from "./UserCard";
+import { useEffect, useState } from "react";
+
+const AllUsers = () => {
+    const [isLoading, setIsLoading] = useState(true);
+    const [Users, setUsers] = useState([]);
+  
+    useEffect(() => {
+      setIsLoading(true);
+      fetchUsers().then((getUsers) => {
```

Comment:
'getUsers' is not the right variable name here. This word should be a noun like 'users'.